WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -196,6 +196,61 @@ def register
     end
   end
 
+  def process_payment
+    competition = competition_from_params
+    if current_user
+      registrations = competition.registrations
+      registration = registrations.find_by_user_id!(current_user.id)
+    end
+    token = params[:stripeToken]
+    Stripe.api_key = ENVied.STRIPE_API_KEY
```

Comment:
I'm still kind of weirded out by this. Would this be more appropriate as a rails intializer? See https://github.com/jfly/worldcubeassociation.org/blob/master/WcaOnRails/config/initializers/delayed_job_config.rb for an example. Docs here http://guides.rubyonrails.org/configuring.html#using-initializer-files.